packages/components/src/utils/property-checkers/check-one-of.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright 2023 by Swiss Post, Information Technology
+ */
```

Comment:
I'm not sure if this is required, since we have a license at every package root containing a copyright notice. Do you have more infos about this?

Reply:
Good question, this is added automatically as soon as I create a new file with intelliJ but I don't know if it is necessary in our case. I will delete that for now.